src/srtctl/core/backend.py

```
@@ -0,0 +1,212 @@
+        for key, value in sorted(config.items()):
+            flag = key.replace("_", "-")
+            if isinstance(value, bool):
+                if value:
+                    lines.append(f"    --{flag} \\")
+            elif isinstance(value, list):
+                lines.append(f"    --{flag} {' '.join(str(v) for v in value)} \\")
+            else:
+                lines.append(f"    --{flag} {value} \\")
+        return lines
+
+    def generate_config_file(self, params: dict = None) -> Path | None:
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)